### ISSUE TITLE
Misleading error message in case of missing quotes

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/python-jsonpreprocessor#94]
I define a string value as composite string in these ways:

```
"composite_string_1" : (1.01)--'${string_01}'--(1.02)--'${string_02}'
"composite_string_2" : ${string_01} -- '${string_01}${string_02}' -- ${string_02}
```

In both cases the quotes around the composite string at the right hand side of the colon are missing. But in case of a composite string (dollar operator expressions together with hard coded string parts) quotes are required. This requirement is not fulfilled. Therefore errors are expected.

The difference between `composite_string_1` and `composite_string_2` is: The value of `composite_string_1` contains dollar operator expressions inside, whereas the value of `composite_string_2` starts and ends with a dollar operator expression. It seems that this difference has an (unwanted) impact on the parser.

The error message for `composite_string_1` is like expected: `Expecting value:`
The error message for `composite_string_2` deviates: `Expecting ',' delimiter`

But the root problem is the same in both cases: missing quotes. Therefore no value can be identified. Therefore the error message also for `composite_string_2` is expected to be: `Expecting value:`

**Comment by HolQue** [comment]
Please refer to rules of value identification in

https://github.com/test-fullautomation/python-jsonpreprocessor/issues/95

**Comment by namsonx** [comment]
Hello Holger,

Thank you for your finding, I will create a pull-request to cover this scenario.

Thank you,
Son

**Comment by namsonx** [comment]
Hello Holger,

I created pull-request #115 for this ticket. The latest commit was merge to my testing branch.

Thank you,
Son

**Comment by HolQue** [comment]
Retest successful.

**Comment by test-fullautomation** [comment]
solved and already merged #147
